model/src/test/kotlin/spdx/SpdxExpressionParserTest.kt

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.spdx
+
+import io.kotlintest.shouldBe
+import io.kotlintest.shouldThrow
+import io.kotlintest.specs.WordSpec
+
+class SpdxExpressionParserTest : WordSpec() {
+    init {
+        "SpdxExpressionParser" should {
+            "parse a license id correctly" {
```

Comment:
For some of the test case names 'correctly' could be replaced with what correctly actually means, e.g. what is asserted...

Reply:
If think the names can be significantly improved, mind making a PR later? I checked and couldn't really think of better names.